src/components/EntityInfo.vue

```diff
@@ -129,70 +127,62 @@
         <v-col cols="12" md="3">
           <dl>
             <!-- Registration Date -->
-            <template v-if="businessId && isFirm">
+            <template v-if="!!businessId && isFirm">
               <dt class="mr-2">Registration Date:</dt>
               <dd id="entity-business-registration-date">
                 {{ this.dateToPacificDate(getFoundingDate, true) || 'Not Available' }}</dd>
             </template>
 
             <!-- Registration Number -->
-            <template v-if="businessId && isFirm">
+            <template v-if="!!businessId && isFirm">
               <dt class="mr-2">Registration Number:</dt>
               <dd id="entity-business-registration-number">{{ getIdentifier || 'Not Available' }}</dd>
             </template>
 
             <!-- Business Number -->
-            <template v-if="businessId">
+            <template v-if="!!businessId">
               <dt class="mr-2">Business Number:</dt>
               <dd id="entity-business-number">{{ getBusinessNumber || 'Not Available' }}</dd>
             </template>
 
             <!-- Incorporation Number -->
-            <template v-if="businessId && !isFirm">
+            <template v-if="!!businessId && !isFirm">
               <dt class="mr-2">Incorporation Number:</dt>
               <dd id="entity-incorporation-number">{{ getIdentifier || 'Not Available' }}</dd>
             </template>
 
             <!-- NR Number -->
-            <template v-if="nameRequestNumber">
+            <template v-if="!!nameRequestNumber">
               <dt class="mr-2">Name Request Number:</dt>
               <dd id="nr-number">{{ nameRequestNumber }}</dd>
             </template>
 
             <!-- Email -->
-            <template v-if="businessId">
+            <template v-if="!!businessId">
               <dt class="mr-2">Email:</dt>
               <dd
                 id="entity-business-email"
-                :class="isAllowed(AllowableActions.EDIT_BUSINESS_PROFILE) ? 'cursor-pointer' : 'pointer-events-none'"
```

Comment:
This is now always true, so I just made it a static class attribute.

Same below for Phone.